modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -54,6 +54,11 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # Ensures that the file is a string file path. Otherwise, default to pandas.
         filepath_or_buffer = cls.get_path_or_buffer(filepath_or_buffer)
         if isinstance(filepath_or_buffer, str):
+            if "*" not in filepath_or_buffer:
+                warnings.warn(
+                    "Shell-style wildcard '*' must be in the filename in order to read multiple "
+                    f"files at once; passed filename: '{filepath_or_buffer}'"
```

Comment:
We can read multiple files without `*` for now (if reading happens  from directory).  Looks like comment/condition is incorrect.

For current master:
```python
import modin.config as cfg
cfg.IsExperimental.put(True)

import modin.experimental.pandas as pd
df = pd.read_csv_glob("s3://modin-datasets/testing/multiple_csv/") # successfully works
```

Reply:
Good point! I'll update the condition

Reply:
@prutskov the condition is updated